Docker_Windows.md

```diff
@@ -26,31 +26,33 @@
 12. Inside the downloaded folder, copy `.env.example` and name the copy `.env`
 13. Open `.env` with notepad by right-clicking it and selecting _Open With_ -> _More apps_ -> _Notepad_.
```

Comment:
In case you are going to edit this file here anyways, could you please fix the capitalization:
```suggestion
13. Open `.env` with Notepad by right-clicking it and selecting _Open With_ -> _More apps_ -> _Notepad_.
```